ocaml/xapi/xapi_globs.ml

```diff
@@ -454,7 +454,7 @@ let rpu_allowed_vm_operations = [
   `update_allowed_operations;
 ]
 
-let rpu_allowed_vdi_operations = [
+let pre_ely_vdi_operations = [
```

Comment:
I think we should restore the exact set of allowed operations that my PR removed: https://github.com/xapi-project/xen-api/pull/3122/files
That means that this list should not include `force_unlock` and `blocked`.